python/cuml/test/dask/test_nearest_neighbors.py

```diff
@@ -204,7 +213,7 @@ def test_default_n_neighbors(cluster):
 
         wait(X_cudf)
 
-        cumlModel = daskNN(verbose=0)
+        cumlModel = daskNN(verbose=1, streams_per_handle=5)
```

Comment:
minutia but it would probably be nice if one bit tests a number of streams_per_handle that is not 5